packages/postgres-database/src/simcore_postgres_database/migration/versions/af5de00bf4cf_.py

```diff
@@ -0,0 +1,53 @@
+"""empty message
+
+Revision ID: af5de00bf4cf
+Revises: 71ea254837b0
+Create Date: 2023-06-19 08:41:09.835411+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "af5de00bf4cf"
+down_revision = "71ea254837b0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "projects_comments",
+        sa.Column("comment_id", sa.BigInteger(), autoincrement=True, nullable=False),
+        sa.Column("project_uuid", sa.String(), nullable=False),
+        sa.Column("user_id", sa.BigInteger(), nullable=False),
+        sa.Column("content", sa.String(), nullable=True),
+        sa.Column(
+            "created_at",
+            sa.DateTime(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated_at",
```

Comment:
this will not auto-update unless you use a trigger here.
you can find an example in [https://github.com/ITISFoundation/osparc-simcore/blob/c8dee4b88919d63d6139653839febcba933b2546/packages/postgres-database/src/simcore_postgres_database/migration/versions/e0a2557dec27_add_services_limitations.py](https://github.com/ITISFoundation/osparc-simcore/blob/c8dee4b88919d63d6139653839febcba933b2546/packages/postgres-database/src/simcore_postgres_database/migration/versions/e0a2557dec27_add_services_limitations.py)

Reply:
yes you are right, as there is just one db endpoint for updating, I have it in the code like this:
```
projects_comments.update()
                .values(
                    project_uuid=project_uuid,
                    content=content,
                    modified=func.now(),
                )
```
which should do its job.